client/js/navigation/BottomNavigationLayout.js

```diff
@@ -1 +1,34 @@
 // bottom navigation
```

Comment:
remove this comment since it is redundant. Your file name already says BottomNavigation.